hw/ip/rstmgr/doc/theory_of_operation.md

```diff
@@ -151,6 +151,12 @@ Parameter                   | Default       | Description
 
 * [Interface Tables](../../../top_earlgrey/ip/rstmgr/data/autogen/rstmgr.hjson#interfaces)
 
+## Reset Indications for Alert Handler
```

Comment:
Move it after Reset Consistency Checks and make it ### ?

Reply:
done